### PR TITLE
Add proxy support, changed all ajax requests to target full url of 

### PR DESCRIPTION
**Pull request by timothydadams**
client, add cors support on server side.

@JasperC1999 @jayrbarr

### CODE REVIEW

File: client/index.jsx [hunk 1]
```diff
@@ -2,16 +2,25 @@ import React from 'react';
...
 import Details from './components/details.jsx';
 import ImageSelector from './components/imgSelector.jsx';
+import url from './components/apiUrl.js';
 
 
 library.add(faListUl, faMapMarkerAlt, faChevronDown, faRetweet, faTruck);
 
+const GlobalStyle = createGlobalStyle`
+  @import url('https://fonts.googleapis.com/css?family=Montserrat');
+  body {
```
Comment (jayrbarr) [inline client/index.jsx:18]: Nice import! I like it.

File: server/server.js [hunk 2]
```diff
@@ -11,6 +13,7 @@ app.use(bodyParser.json());
 
 app.use(express.static('public'));
 
+app.use(cors());
```
Comment (jayrbarr) [inline server/server.js:17]: This works for cors, but it opens it to all origins. The object in the function call parameter can set the origin(s) to specific URL(s). No need to change. Just thought you'd like to know.

Comment (timothydadams) [reply server/server.js:17]: oh good point.  I can set the origin specifically to localhost instead of opening the entire flood gate.  Thanks for the tip!

Comment (timothydadams) [reply server/server.js:17]: done.

**Pull request merged**